docs/oracle-demo.md

```diff
@@ -80,16 +80,130 @@ Finally, certify the key for use by the connections that request that:
 aea issue-certificates
 ```
 
+### Create the oracle client AEA
+
+From a new terminal (in the same top-level directory), fetch the AEA that will deploy the oracle client contract and call the function that requests the coin price from the oracle contract.
+
+``` bash
+aea fetch fetchai/coin_price_oracle_client:0.1.0
+cd coin_price_oracle_client
+aea install
+```
+
+<details><summary>Alternatively, create from scratch.</summary>
+<p>
+
+Create the AEA that will deploy the contract.
+
+``` bash
+aea create coin_price_oracle
+cd coin_price_oracle
+aea add connection fetchai/http_client:0.16.0
+aea add connection fetchai/ledger:0.12.0
+aea add skill fetchai/simple_oracle_client:0.3.0
+aea install
+aea config set agent.default_connection fetchai/ledger:0.12.0
+```
+
+Then update the agent configuration with the default routing:
+``` bash
+aea config set --type dict agent.default_routing \
+'{
+"fetchai/contract_api:0.10.0": "fetchai/ledger:0.12.0",
+"fetchai/http:0.11.0": "fetchai/http_client:0.16.0",
+"fetchai/ledger_api:0.9.0": "fetchai/ledger:0.12.0"
+}'
+```
+
+Change the default ledger:
+``` bash
+aea config set agent.default_ledger ethereum
+```
+
+</p>
+</details>
+
+Create the private key for the oracle client AEA. Generate and add a key for Ethereum use:
+
+``` bash
+aea generate-key ethereum
+aea add-key ethereum ethereum_private_key.txt
+```
```

Comment:
By default this is enough:
```suggestion
``` bash
aea generate-key ethereum
aea add-key ethereum
```
```